spk/readarr/src/service-setup.sh

```diff
@@ -0,0 +1,35 @@
+
+# Readarr service setup
+READARR="${SYNOPKG_PKGDEST}/share/Readarr/bin/Readarr"
+
+# Readarr uses custom Config and PID directories
+HOME_DIR="${SYNOPKG_PKGVAR}"
+CONFIG_DIR="${HOME_DIR}/.config"
```

Comment:
IMO rather than guessing where the config is going to end up, you should pass the `-data=` argument to Readarr to force it to put it in the location you want

Reply:
@ta264 is this for all *arr packages? _(I have to check the command line parameters...)_

Reply:
Yes the \*arrs all support `-nobrowser -data=`

Reply:
regarding the [radar wiki](https://wiki.servarr.com/radarr/appdata-directory#Argument) this is only required when running multiple instances of the same service.

Reply:
> regarding the [radar wiki](https://wiki.servarr.com/radarr/appdata-directory#Argument) this is only required when running multiple instances of the same service.

Yes but that doesn't restrict it from being used for a single instance. And as Ta said, rather than guessing at things, just set it.

Reply:
just wondered why none of the *arr package here does use this (not even radarr udpated by ta264 in #4464) 😄 
So I will update all *arr packages next.

Reply:
I was younger and less wise :D

The radarr improvements I'm working on do include that argument

Reply:
We have a lot of copy&paste in the service-setup scripts of the *arr packages that need a brush up - so I will not only prevent the package downgrade but also add the -data parameter to the service command.

Reply:
And redarding the *arr tools I am still a noob

Reply:
> We have a lot of copy&paste in the service-setup scripts of the *arr packages that need a brush up - so I will not only prevent the package downgrade but also add the -data parameter to the service command.

I'm happy to take on that responsibility. Is the change that needs to be effected across `*arr` (Sonarr, Radarr, Lidarr, Prowlarr) packages the addition of the string "`-nobrowser -data=${CONFIG_DIR}`" at the end of the `SERVICE_COMMAND` parameter?